app/scripts/controllers/auth.js

```diff
@@ -0,0 +1,94 @@
+'use strict';
+
+/**
+ * @ngdoc function
+ * @name todoApp.controller:AboutCtrl
+ * @description
+ * # AboutCtrl
+ * Controller of the todoApp
+ */
+angular.module('todoApp')
+  .controller('authCtrl', function ($scope, $window, User, ngNotify) {
+
+    //Initializing our user object
+    $scope.user = {};
+
+    //Initializing our user token
+    $scope.currentSavedToken = localStorage.getItem("token");
```

Comment:
This variable is okay, and has no issues. I think it would be better to standardize your scope "token" variable across all controllers, that way you can count on $scope.token always getting your token without having to worry about what it's named for each specific controller. This applies to the main control as well.